src/modules/item/sagas.ts

```diff
@@ -381,10 +381,14 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
         if (location.pathname === locations.itemEditor()) {
           if (collectionId) {
             const paginationData: ItemPaginationData = yield select(getPaginationData, collectionId)
-            yield put(fetchCollectionItemsRequest(collectionId, { page: paginationData.currentPage, limit: paginationData.limit }))
+            if (paginationData) {
+              yield put(fetchCollectionItemsRequest(collectionId, { page: paginationData.currentPage, limit: paginationData.limit }))
+            }
           } else {
             const paginationData: ItemPaginationData = yield select(getPaginationData, address)
-            yield put(fetchItemsRequest(address, { page: paginationData.currentPage, limit: paginationData.limit }))
+            if (paginationData) {
```

Comment:
if there's no pagination, could we pass the DEFAULT_PAGE and the DEFAULT_PAGE_SIZE?

yield put(fetchCollectionItemsRequest(collectionId, { page: paginationData.currentPage || DEFAULT_PAGE, limit: paginationData.limit || DEFAULT_PAGE_SIZE}))